src/сomponents/App/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { render } from "react-dom";
```

Comment:
Why do you need a render here?
It should be used once, when you render app to the DOM (`src/index.js`)

Reply:
Removed, thank you! I forgot to clear it after refactoring and moving the render logic to`src/index.js`